components/clp-py-utils/clp_py_utils/sql_adapter.py

```
@@ -139,7 +139,8 @@ def _create_mysql_connection(
                 logging.exception("Database access denied.")
             elif err.errno == errorcode.ER_BAD_DB_ERROR:
                 logging.exception(
-                    f'Specified database "{self.database_config.name}" does not exist.'
+                    f'Specified database "{self.database_config.names[ClpDbNameType.CLP]}" does '
+                    "not exist."
```

Ruff (0.16.1)

[warning] Logging statement uses f-string

(G004)